recipes/btyacc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(test_package C)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
+
+find_program(BTYACC_EXECUTABLE btyacc)
+if(NOT BTYACC_EXECUTABLE)
+  message(FATAL_ERROR "BTYACC_EXECUTABLE is required (Value: ${BTYACC_EXECUTABLE})")
+endif()
+
+set(bin "${PROJECT_BINARY_DIR}")
+add_custom_command(
+    OUTPUT main.c
+    COMMAND "${BTYACC_EXECUTABLE}" -b y "${PROJECT_SOURCE_DIR}/test.y"
```

Comment:
seems like, `BTYACC_EXECUTABLE` cannot be run in case of cross-building
or `test_package` should use `test_type = "build_requires"`...

Reply:
How come it's not an issue in #8951 then? The executable is run pretty much the same.

Reply:
in #8951, executable is not run for cross-building:
https://github.com/conan-io/conan-center-index/pull/8951/files#diff-8bc7fcecacd719fd58e06fbe2069d5f578e730dcbeabf6932e2df28fc69ff270R10

in your case, it's triggered always here:
https://github.com/conan-io/conan-center-index/pull/8947/files#diff-27ab74ef8078f9a4bf308fb9e5d5a4b4340078890b64bcc8e74d44c442651f40R18

(CMake triggers generation of .c file with custom command)

Reply:
Ooh, duh. Totally missed that.

Reply:
Addressed in https://github.com/conan-io/conan-center-index/pull/8947/commits/26736e3cddca085988fcd6d14ecfc888db771f22. I didn't use `test_type`, because found basically no documentation on it.

Reply:
thanks, reported https://github.com/conan-io/docs/issues/2367